war/pom.xml

```diff
@@ -645,7 +647,7 @@ THE SOFTWARE.
       <plugin>
         <groupId>org.eclipse.jetty.ee9</groupId>
         <artifactId>jetty-ee9-maven-plugin</artifactId>
-        <version>12.0.16</version>
+        <version>12.0.17</version>
```

Comment:
Should we stay with `12.0.16` or go to `12.0.18`?
c.f. https://groups.google.com/g/jenkinsci-dev/c/vm4XhQ7FhqY/m/ijXXUTdYBAAJ?utm_medium=email&utm_source=footer